chrome/manifest.dev.json

```diff
@@ -30,6 +30,6 @@
     "page": "options.html",
     "chrome_style": true
   },
-  "permissions": [ "contextMenus", "management", "notifications", "tabs", "storage", "https://github.com/*", "http://localhost:5000/api/v1/*" ],
-  "content_security_policy": "default-src 'self' http://localhost:3000/ ; font-src http://localhost:3000/js/; script-src 'self' http://localhost:3000 https://www.google-analytics.com/ 'unsafe-eval'; connect-src http://localhost:3000 http://localhost:5000/api/v1/; style-src * 'unsafe-inline' 'self' blob:; img-src 'self' data:;"
+  "permissions": [ "contextMenus", "management", "notifications", "tabs", "storage", "http://*/" ],
```

Comment:
Shouldn't this be `http://*` (without trailing slash)? (Otherwise, `https://localhost:5000/api/v1/bookmarks` doesn't match.)

And also include `https://*`?